src/static/js/ours/client.js

```diff
@@ -125,8 +125,8 @@ CODALAB.api = {
     /*---------------------------------------------------------------------
          Datasets
     ---------------------------------------------------------------------*/
-    get_datasets: function (query, type) {
-        return CODALAB.api.request('GET', URLS.API + `datasets/?q=${query || ''}&type=${type || ''}`)
+    get_datasets: function (query, type, page) {
+        return CODALAB.api.request('GET', URLS.API + `datasets/?q=${query || ''}&page=${page || 1}&type=${type || ''}`)
```

Comment:
we should be using `$.param()` or what-have-you, like you can see here: https://github.com/codalab/chahub/blob/develop/src/static/js/ours/client.js#L17

Reply:
why not
```javascript
    get_datasets: function (filters) {
        return CODALAB.api.request('GET', `${URLS.API}datasets/`, filters)
    },
```
?